### PR TITLE
IBM Copyright Additions

### PR DESCRIPTION
**Pull request by chrisdennis**

### CODE REVIEW

File: build-logic/src/main/java/org/ehcache/build/conventions/DeployConvention.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public void apply(Project project) {
         pom.developers(devs -> devs.developer(dev -> {
           dev.getName().set("Terracotta Engineers");
           dev.getEmail().set("tc-oss@softwareag.com");
```
Comment (aurbroszniowski) [inline build-logic/src/main/java/org/ehcache/build/conventions/DeployConvention.java:78]: shouldn't tc-oss@softwareag.com be removed?

**Comment by cljohnso** [comment]
Tons of references to "Software AG" in `README.md` files.

**Comment by chrisdennis** [comment]
> Tons of references to "Software AG" in `README.md` files -- including URLs.

Right... there is lots of language referencing licensing and usage of the Docker images. I've left that all in place for the moment until there is guidance on what it should be replaced with.

File: build-logic/src/main/java/org/ehcache/build/conventions/DeployConvention.java (hunk 1 above)
Comment (chrisdennis) [reply build-logic/src/main/java/org/ehcache/build/conventions/DeployConvention.java]: Replaced with `tc-oss-dg@ibm.com`

**Review by cljohnso — COMMENTED** [review]
Looks good other that what's been mentioned.

**Review by cljohnso — APPROVED** [review]
Accepting the update for @aurbroszniowski comment.

**Pull request merged**